Header header
uint32[] proximities
